src/app/pages/credentials/certificates-dash/certificates-dash.component.ts

```diff
@@ -390,15 +396,17 @@ export class CertificatesDashComponent implements OnInit {
     }, 200);
   }
 
+  private signCsrFieldConfSelect: FormSelectConfig = {
```

Comment:
Please name as `unsignedCertificatesField` or `unsignedCsrSelectField` instead of `signCsrFieldConfSelect`